static/styles/message.css

```diff
@@ -0,0 +1,120 @@
+$first_column_width: 46px;
+
+.message_row {
+    &.include-sender {
+        .messagebox .messagebox-content {
+            grid-template-rows: 25px repeat(3, auto);
```

Comment:
`25px` so that we can fit any text here.

Reply:
I don't understand this comment at all.

Reply:
I mean text in different languages can go take a lot of height regardless of line-height or font-size. `25px` seems safe for that.